docs/web/package-structure.md

```diff
@@ -1 +1,352 @@
-# Package Structure
+# Define a Structure
+
+This document explains what elements should be included when configuring a Front Package and what functions each element performs.
+
+## Directory Structure
+
+The package structure is as follows:
+
+```plaintext
+├── components
+│    ├── button
+│    │    ├── mod.rs
+│    ├── checkbox
+│    │    ├── mod.rs
+│    ├── selectbox
+│          ├── mod.rs
+├── layouts
+│    ├── root_layout
+│          ├── header.rs
+│          └── mod.rs
+│          └── footer.rs
+│          └── side_bar.rs
+├── pages
+│    ├── main
+│    │    ├── controller.rs
+│    │    └── i18n.rs
+│    │    └── mod.rs
+│    │    └── page.rs
+│    └── create
+│          ├── controller.rs
+│          └── i18n.rs
+│          └── mod.rs
+│          └── page.rs
+├── services
+│    ├── user_service.rs
+│    └── popup_service.rs
```

Comment:
we don't recommend to implement popup_service separately. instead of using dioxus_popup.

Reply:
That part was an example part while talking about service, so it might be confusing. I'll fix it.